src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -1,74 +1,80 @@
 <template>
```

Comment:
Most of the changes here in the template are alignment only. You can ignore these and see a simpler diff by going to the ⚙️ icon above and selecting "Hide whitespace".